railgun/test/integration/suite_test.go

```diff
@@ -121,6 +134,21 @@ func TestMain(m *testing.M) {
 		builder = builder.WithExistingCluster(cluster)
 	}
 
+	fmt.Println("INFO: configuring kubernetes cluster")
+	if clusterVersionStr != "" {
+		clusterVersion, err := semver.Parse(strings.TrimPrefix(clusterVersionStr, "v"))
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Error: invalid %s provided (%s): %s", ClusterVersionEnvVar, clusterVersionStr, err.Error())
+			os.Exit(31)
+		}
+		cluster, err := kind.NewBuilder().WithClusterVersion(clusterVersion).Build(ctx)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Error: failed to build kind cluster with version %s: %s", clusterVersion, err.Error())
+			os.Exit(32)
```

Comment:
when using semantic exit statuses, consider defining them in one place. Otherwise a future editor will need to grep for `33`, `34`, `35` etc to find the next free status code (and can fail)

Reply:
Yeah, I've been thinking about this over and over and I just hadn't gotten around to it yet. Now is a good a time as any.

Reply:
Resolved via c5969ec7d94cb399a9067c651faebc4395524f19